src/test/java/org/openrewrite/java/testing/junit5/AddParameterizedTestAnnotationTest.java

```diff
@@ -0,0 +1,435 @@
+                 }
+               }
+              """
+          )
+        );
+    }
+
+    @Test
+    void replacesCsvSource() {
+        rewriteRun(
+          //language=java
+          java(
+            """
+               import org.junit.jupiter.params.provider.CsvSource;
+               import org.junit.jupiter.api.Test;
+               
+               class TestClass {
+                 @Test
+                 @CsvSource({"test@test.com"})
+                 void processUserData(String email) {
+                   System.out.println(email);
+                 }
+               }
+              """,
+            """
+              import org.junit.jupiter.params.ParameterizedTest;
+              import org.junit.jupiter.params.provider.CsvSource;
+              
+              class TestClass {
+                @ParameterizedTest
+                @CsvSource({"test@test.com"})
+                void processUserData(String email) {
+                  System.out.println(email);
+                }
+              }
+              """
+          )
+        );
+    }
+
+    @Test
+    void replacesMethodSource() {
+        rewriteRun(
+          //language=java
+          java(
+            """
+               import org.junit.jupiter.api.Test;
+               import org.junit.jupiter.params.provider.MethodSource;
+               
+               class TestClass {
+                 @Test
+                 @MethodSource()
+                 void foo() {
+                   System.out.println("bar");
+                 }
+               }
+              """,
+            """
+              import org.junit.jupiter.params.ParameterizedTest;
+              import org.junit.jupiter.params.provider.MethodSource;
+                            
+              class TestClass {
+                @ParameterizedTest
+                @MethodSource()
+                void foo() {
+                  System.out.println("bar");
+                }
+              }
+              """
+          )
+        );
+    }
+
+    @Test
+    void addMissingAnnotation() {
+        rewriteRun(
+          //language=java
+          java(
+            """
+             import org.junit.jupiter.params.ParameterizedTest;
+             import org.junit.jupiter.params.provider.ValueSource;
+             import static org.junit.jupiter.api.Assertions.*;
+             
+             class TestClass {
+                @ParameterizedTest
+                @ValueSource(ints = {1, 3, 5, -3, 15, Integer.MAX_VALUE})
+                void testIsOdd(int number) {
+                    assertTrue(number % 2 != 0);
+                }
+             }
+             """,
+            """
+              import org.junit.jupiter.params.ParameterizedTest;
+              import org.junit.jupiter.params.provider.ValueSource;
+              import static org.junit.jupiter.api.Assertions.*;
+              
+              class TestClass {
+                 @ParameterizedTest
```

Comment:
You're inconsistent in your use of indentation in your tests; here you're indenting with three spaces within the class, and four spaces within the method. You'll likely want to standardize that to not trip up the autoformat.
```suggestion
                  @ParameterizedTest
```